Logger/LogFactory.cs

```diff
@@ -2,10 +2,24 @@
 {
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? _pathName;
+
+        public BaseLogger? CreateLogger(string className)
         {
-            
-            return null;
+            if (_pathName == null)
```

Comment:
```suggestion
            if (_pathName is null)
```
Can use `is` here aswell